ui/admin/app/templates/scopes/scope/policies/new.hbs

```diff
@@ -7,4 +7,25 @@
 <Breadcrumbs::Item
   @text={{t 'resources.policy.titles.new'}}
   @route='scopes.scope.policies.new'
-/>
\ No newline at end of file
+/>
+
+<Rose::Layout::Page as |page|>
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+
+  <page.header>
+    <h2>
```

Comment:
None blocking comment: we could have used the HDS::Text component here, but I am planning a refactor of the pageHeader all together, [here the poc](https://github.com/hashicorp/boundary-ui/pull/2037). So all good :smile: